R/travis-s3.R

```diff
@@ -11,7 +11,7 @@ travis_no_attr <- function(x) {
 new_travis <- function(x, attr, subclass) {
   attr[["names"]] <- names(x)
   attributes(x) <- attr
-  structure(x, class = c(paste0("travis_", subclass), "travis"))
+  suppressWarnings(structure(x, class = c(paste0("travis_", subclass), "travis")))
```

Comment:
What are we suppressing here?

Reply:
I've answered this already some weeks ago.
Can't recall just right now and do not see a warning anymore currently when debugging. Removed it again but I guess it might bite back again at some point. If so, we can return here.

Reply:
If this triggers a warning, we should scout for the core of the problem.